docs/notebooks/README.md

```diff
@@ -0,0 +1,24 @@
+# Using JuPedSim with Jupyter notebooks 
```

Comment:
Can we publish these requirements on the website? Where?

Reply:
Yes, a good idea. 

I don't know yet, but I guess I will move tutorials from analysis to `introduction` or below it.


<img width="235" alt="Screenshot 2020-05-12 02 49 24" src="https://user-images.githubusercontent.com/5772973/81626225-5deb5680-93fb-11ea-9d21-506bbb2e3264.png">

Reply:
These notebooks are not only for _jpsreport_. Why don't we create a separate section on the top level? Then we can create a general tutorial for jpscore and jpsreport.

Reply:
"Introduction" is top level. 
<img width="288" alt="Screenshot 2020-05-12 09 06 32" src="https://user-images.githubusercontent.com/5772973/81649035-f996ba00-942f-11ea-87cb-2ab96af0ba08.png">

Reply:
Moved documentation to top-level for better visibility.

4ba1228